.eslintrc

```diff
@@ -0,0 +1,158 @@
+rules:
+  indent:
+    - 2
+    - 2
+  quotes:
+    - 2
+    - single
+  linebreak-style:
+    - 2
+    - unix
+  semi:
+    - 2
+    - never
+  array-bracket-spacing: 
+    - 2
+    - "never"
+  block-scoped-var: 2
+  brace-style: 
+    - 2
+    - "stroustrup"
+    - 
+      allowSingleLine: true
+  comma-dangle: 
+    - 2
+    - "never"
```

Comment:
I've added `vars-on-top`. I agree, it's a best practice and it's only a pain with long overly complicated functions… so let's just not have those. :stuck_out_tongue_winking_eye: Or at least not optimize for them.

James and I also discussed the trailing commas - it's not a problem with anything newer than IE8 and we're not currently supporting that, so trailing commas *ALWAYS*. :grinning: 

Finally, I think dropping the semicolons is a WTF for most of the team and out of line with most of the community.  We all keep typing semicolons and having to delete them. Further, it really has caused some issues with the javascript concatenation and minification - so… sorry @zanchin  but I think there'll just be a bit more typing.  Luckily eslint can put the semi's in there for you before you commit ;-)

@tortillaj I'm totally in favor of chained syntax being indented, it's way more readable.  My rule is usually one call goes on the same line, two get wrapped and indented.

```` javascript
// This is fine.
foo.bar();
// This gets you beaten.
foo.bar().baz();
// This is fine.
foo
  .bar()
  .baz();